token-metadata/js/README.md

```diff
@@ -0,0 +1,13 @@
+# Token Metadata
+
+## Installation
+
+```shell
+npm install @metaplex-foundation/mpl-token-metadata --save
+```
+
+This will use the new auto-generated API. If you want to use the old deprecated API, you can use the following:
+
+```shell
+npm install @metaplex-foundation/mpl-token-metadata@^1.2 --save
+```
```

Comment:
Should add something along the lines of the below:

~~~

In order to start using the new API but still have access to the deprecated one please do the following:

```sh
npm install @metaplex-foundation/mpl-token-metadata@~2.0 --save
```

```sh
import { deprecated } from @metaplex-foundation/mpl-token-metadata
```

Reply:
Do we really want to do that though if `2.0` is not really working with the new API due to `spl-token` being forced to be the old version (0.1)?

Reply:
It's just to explain an option for people ... they could use different spl-tokens in the same app .. it's just an issue once you bundle in the browser potentially afaik.

But I'm fine either way .. it's mainly also so we can link people to it if we get asked.

Reply:
Got it, I'll add now :)